scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -32,8 +32,6 @@ oc login -u developer -p password@123 --insecure-skip-tls-verify
 # Check login user name for debugging purpose
 oc whoami
 
-source ./scripts/openshiftci-config.sh
```

Comment:
/hold

Correct me if I'm wrong @anandrkskd and @ritudes, but I think this script is used for interop tests. 
For our own periodic tests, I agree it certainly makes sense to catch issues with the staging registry early on, but wouldn't we want to rely on a more stable registry for interop tests?

Reply:
Personally, I would go the other way: testing the compatibility with registry during the night, and have a stable env for PRs.

But I defer to the QE decision

Reply:
> Personally, I would go the other way: testing the compatibility with registry during the night, and have a stable env for PRs.
> 
Yup, I also agree with this (at least for tests that are under our control, like our nightlies). I am just not sure to what extent interop tests would need a stable Devfile registry.
From my understanding, those "interoperability" tests are supposed to test odo against unreleased versions of OpenShift, so I thought it would make more sense to provide them with a stable Devfile registry.

Reply:
>Correct me if I'm wrong (@anandrkskd and @ritudes), but I think this script is used for interop tests.

Yes Armel this script if used for interop testing.

> From my understanding, those "interoperability" tests are supposed to test odo against unreleased versions of OpenShift, so I thought it would make more sense to provide them with a stable Devfile registry.

True, I think it would be better choice to use stable devfile registry for interOP tests as those tests are testing odo with unreleased version of OpenShift.